opplast/upload.py

```diff
@@ -33,13 +35,15 @@ def upload(self, meta: dict) -> (bool, str):
             "title": "my title",
             "description": "my description",
             "thumbnail": "path/to/thumbnail.jpg"
+            "tags": ["list","of","tags"](LIST OF STRINGS, WITH INTS IT DOESN'T WORK)
         }
 
         Returns if the video was uploaded and the video id.
         """
         video = meta.get("file")
         title = meta.get("title")
         description = meta.get("description")
+        tags = set(meta.get("tags"))

```

Comment:
I think that i should convert the tags to a set, only if there are tags sent, I don't know how to modify a pr sry